[ERROR] 2015-08-20 11:37:21,460 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.exceptions.MySQLIntegrityConstraintViolationException: Duplicate entry '1' for key 'PRIMARY'
SQL:insert into iTask_tag(id, content, create_time, user_id) values(1, '工作', 1402017888889, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.exceptions.MySQLIntegrityConstraintViolationException: Duplicate entry '1' for key 'PRIMARY'
SQL:insert into iTask_tag(id, content, create_time, user_id) values(1, '工作', 1402017888889, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.exceptions.MySQLIntegrityConstraintViolationException: Duplicate entry '1' for key 'PRIMARY'
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:931)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2934)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,949 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(1, '工作', 1402017888889, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(1, '工作', 1402017888889, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,952 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(3, '安卓项目', 1402024268990, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(3, '安卓项目', 1402024268990, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,955 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(5, '运动', 1402030411067, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(5, '运动', 1402030411067, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,958 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(6, '学习', 1402030506712, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(6, '学习', 1402030506712, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,962 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(7, '纸质书', 1402030521000, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(7, '纸质书', 1402030521000, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,965 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(8, '电子书', 1402030527119, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(8, '电子书', 1402030527119, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,967 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(9, '成长', 1406764942747, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(9, '成长', 1406764942747, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,969 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(10, '待办事项', 1406781408484, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(10, '待办事项', 1406781408484, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,971 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(11, '有声小说', 1407322766502, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(11, '有声小说', 1407322766502, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,973 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(12, '其他项目', 1410241734992, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(12, '其他项目', 1410241734992, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,976 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(13, '纪录片', 1410325001136, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(13, '纪录片', 1410325001136, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,979 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(14, '结婚相关', 1411296882815, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(14, '结婚相关', 1411296882815, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
[ERROR] 2015-08-20 12:06:58,982 method:com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:20)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(15, '公开课', 1411706103510, 1)
com.newland.bg.commontools.exception.NewlandSQLException: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
SQL:insert into iTask_tag(id, content, create_time, user_id) values(15, '公开课', 1411706103510, 1)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:400)
	at com.lynn.iTaskServer.dao.TaskDao.saveTag(TaskDao.java:18)
	at com.lynn.iTaskServer.core.TaskServer.task(TaskServer.java:38)
	at com.lynn.iTaskServer.core.TaskServer.main(TaskServer.java:78)
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Out of range value for column 'create_time' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2932)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1616)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1708)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3255)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1293)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1566)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1480)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:1465)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:105)
	at com.newland.bg.commontools.db.JDBCUtil.execute(JDBCUtil.java:396)
	... 3 more
